src/source-pcap-file-helper.c

```diff
@@ -196,39 +221,40 @@ TmEcode InitPcapFile(PcapFileFileVars *pfv)
     pfv->datalink = pcap_datalink(pfv->pcap_handle);
     SCLogDebug("datalink %" PRId32 "", pfv->datalink);
 
+    if (!PeekFirstPacketTimestamp(pfv))
+        SCReturnInt(TM_ECODE_FAILED);
+
     DecoderFunc temp;
     TmEcode validated = ValidateLinkType(pfv->datalink, &temp);
     SCReturnInt(validated);
 }
 
-TmEcode ValidateLinkType(int datalink, DecoderFunc *decoder)
+TmEcode ValidateLinkType(int datalink, DecoderFunc *DecoderFn)
```

Comment:
style? parameter names are lowercase?

Reply:
Func pointers should be in this notation.